v8isolate.cc

```diff
@@ -20,6 +20,18 @@ V8Isolate::V8Isolate() {
   isolate_ = v8::Isolate::New(create_params);
 }
 
+V8Isolate::V8Isolate(const char* snapshot_data) {
```

Comment:
snapshot_js_str is fine with me, even snapshot_js would work since js is implicitly a string.